src/map/map_draw.cpp

```diff
@@ -329,25 +330,20 @@ void CViewport::DrawMapBackgroundInViewport(const fieldHighlightChecker highligh
 			Map.TileGraphic->DrawFrameClip(tile, dx, dy);
 #ifdef DEBUG
 			// AStar passability overlay
-			if (CViewport::isGridEnabled()) {
-				for (int i = 0; i < graphicTileOffset; i++) {
-					for (int j = 0; j < graphicTileOffset; j++) {
```

Comment:
I deleted them by mistake - I didn't recognize right away that Tim implemented highlighting not only for tiles but also for  subtiles (in cases when graphic tile is larger when logical tile). So it will be necessary to return them. For now I've changed the PR status to draft.

Reply:
With a little bit of modification, I've reverted highlighting for subtiles. 
This way it will take a bit longer to draw, but I don't think it's critical considering that this function is used only for debugging.  But it is noticeably more readable.